### ISSUE TITLE
Git Bash denon: command not found

### ISSUE THREAD

**Issue by ngtantai1001** [denosaurs/denon#62]
```bash
Administrator@TAIDEPTRAI-PC MINGW64 ~
$ deno install --allow-read --allow-run --allow-write -f --unstable https://deno.land/x/denon/denon.ts
Download https://deno.land/x/denon/denon.ts
Download https://deno.land/x/denon/deps.ts
...
Compile https://deno.land/x/denon/denon.ts
✅ Successfully installed denon
C:\Users\Administrator\.deno\bin\denon.cmd

Administrator@TAIDEPTRAI-PC MINGW64 ~
$ denon
bash: denon: command not found

Administrator@TAIDEPTRAI-PC MINGW64 ~
$
```

After this.
Close and open Git bash. Still not working.
Powershell works well.

**Comment by Qu4k** [comment]
Hi.  Is `C:\Users\Administrator\.deno\bin` in your path?

**Comment by ngtantai1001** [comment]
> Hi. Is `C:\Users\Administrator\.deno\bin` in your path?

Yes. 

Tried with this command `denon.cmd` and it works well.

```bash
Administrator@TAIDEPTRAI-PC MINGW64 /d/Code/hack-shopee-new (master)
$ denon.cmd
```

**Comment by Qu4k** [comment]
This happens even on other tools installed using the `deno install` command or only with denon?

**Comment by lengfangbing** [comment]
I got the same question
![image](https://user-images.githubusercontent.com/46396510/84155935-82d6f600-aa9b-11ea-8551-8b949cf12b67.png)
Should I write the export in ~/.zshrc ?

**Comment by Qu4k** [comment]
Please refer to script installer documentation in the deno website. https://deno.land/manual/tools/script_installer

tl; dr: `$HOME/.deno/bin` to your path 🌟

**Comment by lengfangbing** [comment]
> Please refer to script installer documentation in the deno website. https://deno.land/manual/tools/script_installer
> 
> tl; dr: export `$HOME/.deno/bin` to your path 🌟

Yes! Got it , thank you ^_^

**Comment by dstadz** [comment]
![Capture](https://user-images.githubusercontent.com/16406971/84172143-83867280-aa30-11ea-9d87-7b2173c22f6c.PNG)

I followed the instructions and added to Path, but it's still not working?

**Comment by eliassjogreen** [comment]
You might have to refresh your environment variables, do this by restarting your terminal (works most of the time)

**Comment by dstadz** [comment]
Restarted terminal, restarted vsCode, still 

bash: denon: command not found
